common/src/lib.rs

```diff
@@ -598,56 +528,15 @@ pub fn waiting_init_take_messages(dest_prog_id: H256) -> Vec<H256> {
     messages.unwrap_or_default()
 }
 
-pub fn code_exists(code_hash: H256) -> bool {
-    sp_io::storage::exists(&code_key(code_hash, CodeKeyPrefixKind::Code))
-}
-
 pub fn reset_storage() {
     sp_io::storage::clear_prefix(STORAGE_PROGRAM_PREFIX, None);
     sp_io::storage::clear_prefix(STORAGE_PROGRAM_PAGES_PREFIX, None);
     sp_io::storage::clear_prefix(STORAGE_MESSAGE_PREFIX, None);
-    sp_io::storage::clear_prefix(STORAGE_CODE_PREFIX, None);
-    sp_io::storage::clear_prefix(STORAGE_ORIGINAL_CODE_PREFIX, None);
     sp_io::storage::clear_prefix(STORAGE_WAITLIST_PREFIX, None);
     sp_io::storage::clear_prefix(GAS_VALUE_PREFIX, None);
-}
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    fn get_active_program(id: H256) -> Option<ActiveProgram> {
-        get_program(id).and_then(|p| p.try_into().ok())
-    }
 
-    #[test]
-    fn program_decoded() {
-        sp_io::TestExternalities::new_empty().execute_with(|| {
-            let code =
-                hex_literal::hex!("0061736d01000000020f0103656e76066d656d6f7279020001").to_vec();
-            let code_id = CodeId::generate(&code).into_origin();
-            let code = Code::try_new(
-                code,
-                1,
-                None,
-                wasm_instrument::gas_metering::ConstantCostRules::default(),
-            )
-            .unwrap();
-
-            let program_id = H256::from_low_u64_be(1);
-            let program = ActiveProgram {
-                static_pages: 256.into(),
-                persistent_pages: Default::default(),
-                code_hash: code_id,
-                state: ProgramState::Initialized,
-            };
-            set_code(code_id, &code);
-            assert!(get_program(program_id).is_none());
-            set_program(program_id, program.clone(), Default::default());
-            assert_eq!(get_active_program(program_id).unwrap(), program);
-
-            // Structs can't be equal because original_code is skipped. So we asserting binary code.
-            assert_eq!(get_code(program.code_hash).unwrap().code(), code.code());
-        });
-    }
+    pub const STORAGE_CODE_PREFIX: &[u8] = b"g::code::";
+    pub const STORAGE_ORIGINAL_CODE_PREFIX: &[u8] = b"g::code::orig";
+    sp_io::storage::clear_prefix(STORAGE_CODE_PREFIX, None);
+    sp_io::storage::clear_prefix(STORAGE_ORIGINAL_CODE_PREFIX, None);
```

Comment:
Yes, it should but after the next runtime upgrade when this PR is merged. In other case we would have some data in the storage.

I'll create an issue in order not to forget about this.

Reply:
> Yes, it should but after the next runtime upgrade when this PR is merged. In other case we would have some data in the storage.
> 
> I'll create an issue in order not to forget about this.

Better to refer it before this PR merged to prevent forgiving about it